enterprise/server/remote_execution/uffd/uffd.go

```diff
@@ -1,13 +1,118 @@
+//go:build linux
```

Comment:
not sure if we also need to add a `target_compatible_with` attr on the `go_library` target to make sure we don't try to build this unnecessarily on macOS?